package.json

```diff
@@ -27,31 +27,31 @@
     "semantic-release": "semantic-release"
   },
   "devDependencies": {
-    "@adobe/eslint-config-asset-compute": "^1.3.0",
-    "@semantic-release/git": "^9.0.0",
+    "@adobe/eslint-config-asset-compute": "^1.3.2",
+    "@semantic-release/git": "^9.0.1",
     "conventional-changelog-eslint": "^3.0.9",
-    "dotenv": "^8.2.0",
-    "eslint": "^7.11.0",
+    "dotenv": "^8.6.0",
+    "eslint": "^7.32.0",
     "license-checker": "^25.0.1",
-    "lockfile-lint": "^4.3.7",
-    "mocha": "^8.2.0",
+    "lockfile-lint": "^4.6.2",
+    "mocha": "^8.4.0",
     "mock-fs": "^5.0.0",
-    "nock": "^13.0.11",
+    "nock": "^13.1.3",
     "nyc": "^15.1.0",
-    "semantic-release": "^17.4.4",
-    "sinon": "^10.0.0"
+    "semantic-release": "^17.4.7",
+    "sinon": "^11.1.2"
   },
   "dependencies": {
     "@adobe/aio-lib-files": "^2.2.0",
     "@adobe/asset-compute-commons": "^1.1.2",
-    "@adobe/httptransfer": "^2.0.3",
+    "@adobe/httptransfer": "^2.8.0",
     "clone": "^2.1.2",
     "css-unit-converter": "^1.1.2",
     "data-uri-to-buffer": "^3.0.1",
-    "debug": "^4.3.1",
-    "file-type": "^16.5.0",
+    "debug": "^4.3.2",
+    "file-type": "^16.5.3",
     "fs-extra": "^10.0.0",
-    "graph-data-structure": "^1.15.0",
+    "graph-data-structure": "^2.0.0",
```

Comment:
I wouldn't update the major version of this inside this PR

Reply:
Why? We have no breaking changes to account for. Any concerns?